<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="3" failed="1" ignored="0" total="4" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
    </line>
    <line>
      <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //a[@class='login']]]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: email]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: email]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: passwd]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: passwd]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
    </line>
    <line>
      <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //button[@id='SubmitLogin']]]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
    </line>
    <line>
      <![CDATA[Retrying test verifyLoginWithInvalidCredentials with status FAILURE for the 1 time(s).]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
    </line>
    <line>
      <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //a[@class='login']]]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: email]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: email]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: passwd]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: passwd]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
    </line>
    <line>
      <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //button[@id='SubmitLogin']]]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
    </line>
    <line>
      <![CDATA[Retrying test verifyLoginWithInvalidCredentials with status FAILURE for the 2 time(s).]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
    </line>
    <line>
      <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //a[@class='login']]]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: email]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: email]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: passwd]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: passwd]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
    </line>
    <line>
      <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //button[@id='SubmitLogin']]]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
    </line>
    <line>
      <![CDATA[Retrying test verifyLoginWithInvalidCredentials with status FAILURE for the 3 time(s).]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
    </line>
    <line>
      <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //a[@class='login']]]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: email]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: email]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: passwd]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.id: passwd]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
    </line>
    <line>
      <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //button[@id='SubmitLogin']]]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
    </line>
    <line>
      <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
    </line>
    <line>
      <![CDATA[<a href='C:\Development\Eclipse\ProjectHybridFramework\Automation_POM\src\main\java\com\test\automation\uiAutomation\screenshot\blank_18_01_2018_03_14_03.png'> <img src='C:\Development\Eclipse\ProjectHybridFramework\Automation_POM\src\main\java\com\test\automation\uiAutomation\screenshot\blank_18_01_2018_03_14_03.png' height='100' width='100'/> </a>]]>
    </line>
  </reporter-output>
  <suite name="Regression Suite" duration-ms="17161" started-at="2018-01-18T20:13:47Z" finished-at="2018-01-18T20:14:04Z">
    <groups>
    </groups>
    <test name="HomePageTest" duration-ms="17161" started-at="2018-01-18T20:13:47Z" finished-at="2018-01-18T20:14:04Z">
      <class name="com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials">
        <test-method status="PASS" signature="SetUP()[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="SetUP" is-config="true" duration-ms="9368" started-at="2018-01-18T15:13:47Z" finished-at="2018-01-18T15:13:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUP -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="beforeMethod" is-config="true" duration-ms="6" started-at="2018-01-18T15:13:56Z" finished-at="2018-01-18T15:13:56Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials.verifyLoginWithInvalidCredentials()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="SKIP" signature="verifyLoginWithInvalidCredentials()[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="verifyLoginWithInvalidCredentials" duration-ms="1909" started-at="2018-01-18T15:13:56Z" finished-at="2018-01-18T15:13:58Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [TestAuthentication failed.] but found [Authentication failed.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [TestAuthentication failed.] but found [Authentication failed.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials.verifyLoginWithInvalidCredentials(TC001_VerifyLoginWithInvalidCredentials.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
            </line>
            <line>
              <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //a[@class='login']]]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: email]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: email]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: passwd]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: passwd]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
            </line>
            <line>
              <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //button[@id='SubmitLogin']]]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
            </line>
            <line>
              <![CDATA[Retrying test verifyLoginWithInvalidCredentials with status FAILURE for the 1 time(s).]]>
            </line>
          </reporter-output>
        </test-method> <!-- verifyLoginWithInvalidCredentials -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2018-01-18T15:13:58Z" finished-at="2018-01-18T15:13:58Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials.verifyLoginWithInvalidCredentials()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="SKIP" signature="verifyLoginWithInvalidCredentials()[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="verifyLoginWithInvalidCredentials" duration-ms="1718" started-at="2018-01-18T15:13:58Z" finished-at="2018-01-18T15:14:00Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [TestAuthentication failed.] but found [Authentication failed.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [TestAuthentication failed.] but found [Authentication failed.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials.verifyLoginWithInvalidCredentials(TC001_VerifyLoginWithInvalidCredentials.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:998)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1200)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
            </line>
            <line>
              <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //a[@class='login']]]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: email]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: email]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: passwd]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: passwd]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
            </line>
            <line>
              <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //button[@id='SubmitLogin']]]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
            </line>
            <line>
              <![CDATA[Retrying test verifyLoginWithInvalidCredentials with status FAILURE for the 2 time(s).]]>
            </line>
          </reporter-output>
        </test-method> <!-- verifyLoginWithInvalidCredentials -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="afterMethod" is-config="true" duration-ms="0" started-at="2018-01-18T15:13:58Z" finished-at="2018-01-18T15:13:58Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginWithInvalidCredentials status=SKIP method=TC001_VerifyLoginWithInvalidCredentials.verifyLoginWithInvalidCredentials()[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742] output=Found Element By : By.xpath: //a[@class='login']]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="afterMethod" is-config="true" duration-ms="0" started-at="2018-01-18T15:14:00Z" finished-at="2018-01-18T15:14:00Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginWithInvalidCredentials status=SKIP method=TC001_VerifyLoginWithInvalidCredentials.verifyLoginWithInvalidCredentials()[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742] output=Found Element By : By.xpath: //a[@class='login']]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="beforeMethod" is-config="true" duration-ms="1" started-at="2018-01-18T15:14:00Z" finished-at="2018-01-18T15:14:00Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials.verifyLoginWithInvalidCredentials()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="SKIP" signature="verifyLoginWithInvalidCredentials()[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="verifyLoginWithInvalidCredentials" duration-ms="1666" started-at="2018-01-18T15:14:00Z" finished-at="2018-01-18T15:14:01Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [TestAuthentication failed.] but found [Authentication failed.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [TestAuthentication failed.] but found [Authentication failed.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials.verifyLoginWithInvalidCredentials(TC001_VerifyLoginWithInvalidCredentials.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:998)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1200)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
            </line>
            <line>
              <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //a[@class='login']]]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: email]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: email]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: passwd]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: passwd]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
            </line>
            <line>
              <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //button[@id='SubmitLogin']]]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
            </line>
            <line>
              <![CDATA[Retrying test verifyLoginWithInvalidCredentials with status FAILURE for the 3 time(s).]]>
            </line>
          </reporter-output>
        </test-method> <!-- verifyLoginWithInvalidCredentials -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2018-01-18T15:14:01Z" finished-at="2018-01-18T15:14:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials.verifyLoginWithInvalidCredentials()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="afterMethod" is-config="true" duration-ms="0" started-at="2018-01-18T15:14:01Z" finished-at="2018-01-18T15:14:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginWithInvalidCredentials status=SKIP method=TC001_VerifyLoginWithInvalidCredentials.verifyLoginWithInvalidCredentials()[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742] output=Found Element By : By.xpath: //a[@class='login']]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="verifyLoginWithInvalidCredentials()[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="verifyLoginWithInvalidCredentials" duration-ms="1568" started-at="2018-01-18T15:14:01Z" finished-at="2018-01-18T15:14:03Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [TestAuthentication failed.] but found [Authentication failed.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [TestAuthentication failed.] but found [Authentication failed.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials.verifyLoginWithInvalidCredentials(TC001_VerifyLoginWithInvalidCredentials.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:998)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1200)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
            </line>
            <line>
              <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //a[@class='login']]]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //a[@class='login']]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: email]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: email]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: passwd]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.id: passwd]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
            </line>
            <line>
              <![CDATA[Clicked on: [[ChromeDriver: chrome on WIN8_1 (0d43bab6cffbe72ac23caa232b113d43)] -> xpath: //button[@id='SubmitLogin']]]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //button[@id='SubmitLogin']]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
            </line>
            <line>
              <![CDATA[Found Element By : By.xpath: //*[@id='center_column']/div[1]/ol/li]]>
            </line>
          </reporter-output>
        </test-method> <!-- verifyLoginWithInvalidCredentials -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="afterMethod" is-config="true" duration-ms="421" started-at="2018-01-18T15:14:03Z" finished-at="2018-01-18T15:14:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginWithInvalidCredentials status=FAILURE method=TC001_VerifyLoginWithInvalidCredentials.verifyLoginWithInvalidCredentials()[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742] output=Found Element By : By.xpath: //a[@class='login']]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[<a href='C:\Development\Eclipse\ProjectHybridFramework\Automation_POM\src\main\java\com\test\automation\uiAutomation\screenshot\blank_18_01_2018_03_14_03.png'> <img src='C:\Development\Eclipse\ProjectHybridFramework\Automation_POM\src\main\java\com\test\automation\uiAutomation\screenshot\blank_18_01_2018_03_14_03.png' height='100' width='100'/> </a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials@69ea3742]" name="endTest" is-config="true" duration-ms="469" started-at="2018-01-18T15:14:03Z" finished-at="2018-01-18T15:14:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- com.test.automation.uiAutomation.homepage.TC001_VerifyLoginWithInvalidCredentials -->
    </test> <!-- HomePageTest -->
  </suite> <!-- Regression Suite -->
</testng-results>
